docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md

```diff
@@ -0,0 +1,252 @@
+      * **Quick Start Version**. Enter the version of the Quick Start. Do not change.<br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-param-8.png')} alt="Quick start configuration" style={{border: '1px solid black'}} width="700"/> 
+   1. Under **Permissions**, in **IAM role - optional**, choose the IAM role for CloudFormation to use for all operations performed on the stack. 
+   1. Under **Capabilities and transforms**, select the acknowledgement boxes.<br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-permissions.png')} alt="Create Stack button" style={{border: '1px solid black'}} width="700"/> 
+1. Click **Create Stack**. The stack is created, and the app is installed.
+1. Select an option to start using the app. <br/><img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-finish-installation.png')} alt="App hub page" style={{border: '1px solid black'}} width="700"/>
+   
+
+## Cloud Infrastructure Security for AWS app dashboards​
+
+{@import ../../reuse/filter-dashboards.md}
+
+### Active threats dashboards
+
+The Active Threats dashboards show data on threats that require attention. Review these dashboards to see threats identified in AWS APIs, resources, and storage.
+
+#### Active Threats: AWS APIs
+
+The **Active Threats: AWS APIs** dashboard shows threats identified from AWS APIs. It shows threats count and trend, and threats by resource, actor, events, and geo location. 
+
+<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-active-threats-aws-apis.png')} alt="Active Threats: AWS APIs dashboard" width="600"/>
+
+#### Active Threats: AWS Resources
+
+The **Active Threats: AWS Resources** dashboard shows threats identified in AWS resources such as EC2 and IAMUser. It shows findings by resource, trend, resource type, category, and country. Like the [Risk Overview](#resource-risk-dashboard) dashboard, this dashboard has an **Action Plan** panel so you can access suggested resources that need attention.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-active-threats-aws-resources.png')} alt="Active Threats: AWS APIs dashboard" width="600"/>
+
+#### Active Threats: AWS Storage
+
+The **Active Threats: AWS Storage** dashboard provides threat counts related to AWS S3 buckets. It shows threats count and trend, and threats by resource, actor, and geo location.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-active-threats-aws-storage.png')} alt="Active Threats: AWS Storage dashboard" width="600"/>
+
+### Risk Overview dashboard
+
+The **Risk Overview** dashboard provides a summary of all resources that pose risks in a single dashboard that rolls up the findings from other dashboards. It also shows AWS API events by time, and has an **Action Plan** panel so you can access resources that need attention.
```

Comment:
```suggestion
The **Risk Overview** dashboard provides a summary of all resources that pose risks in a single dashboard that rolls up the findings from other dashboards. It also shows AWS API events by time. Like the [Active Threats: AWS Resources](#active-threats-aws-resources) dashboard, this dashboard has an **Action Plan** panel so you can access resources that need attention.
```

Reply:
@kimsauce, I won't make this change, because it's not needed.